plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -27,7 +27,7 @@ private MLCommonsSettings() {}
     public static final Setting<Integer> ML_COMMONS_SYNC_UP_JOB_INTERVAL_IN_SECONDS = Setting
         .intSetting(
             "plugins.ml_commons.sync_up_job_interval_in_seconds",
-            10,
+            3,
```

Comment:
Reducing the sync-up interval to 3 seconds to catch up load failures sooner?

Reply:
We did some testing reduce to 1 second won't impact cluster performance too much. Frontend needs shorter interval to get latest model state.